tldr

```diff
@@ -231,7 +237,7 @@ path_for_cmd() {
 cache_tldr() {
     contains "$(tar --version | head -n 1)" bsdtar || taropt='--wildcards --overwrite'
     curl -sfL https://github.com/tldr-pages/tldr/tarball/master \
-    | tar xzf - --strip-components=2 -C "$configdir" ${taropt-} '*/pages/'
+    | tar xzf - --strip-components=2 -C "$configdir" "${taropt-}" '*/pages/'
```

Comment:
This may be a bad change, did you test it? The previous version specifically relied upon shell space expansion to break the argument list apart. (See the definition of taropt two lines above.)

Reply:
Hm. My version of tar 
```
tar (GNU tar) 1.32
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
$ equery u app-arch/tar
[ Legend : U - final flag setting for installation]
[        : I - package is installed with flag     ]
[ Colors : set, unset                             ]
 * Found these USE flags for app-arch/tar-1.32:
 U I
 + + acl     : Add support for Access Control Lists
 - - minimal : just install `tar`
 + + nls     : Add Native Language Support (using gettext - GNU locale utilities)
 - - static  : !!do not set this during bootstrap!! Causes binaries to be statically linked instead of dynamically
 + + xattr   : Add support for extended attributes (filesystem-stored metadata)
```
Does complain with the quotes:
```
(repo) $ ./tldr man && rm -rf ~/.config/tldr
tar: unrecognized option '--wildcards --overwrite'
Try 'tar --help' or 'tar --usage' for more information.

man

Format and display manual pages.
...
```

But proceeds nonetheless. I will roll this back and add shellcheck disable comment

Reply:
Be a bit careful of accepting ShellCheck's recommendations. It often gives bad advice, even for things it could statically analyze.